Android.mk

```diff
@@ -66,13 +66,15 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/host/xtest \
 		$(LOCAL_PATH)/ta/storage/include \
 		$(LOCAL_PATH)/ta/storage_benchmark/include \
 		$(LOCAL_PATH)/ta/sha_perf/include \
-		$(LOCAL_PATH)/ta/aes_perf/include
+		$(LOCAL_PATH)/ta/aes_perf/include \
+		$(LOCAL_PATH)/ta/socket/include
 
 # Include configuration file generated by OP-TEE OS (CFG_* macros)
 LOCAL_CFLAGS += -include conf.h
 LOCAL_CFLAGS += -DTA_DIR=\"/system/lib/optee_armtz\"
 LOCAL_CFLAGS += -pthread
 LOCAL_CFLAGS += -g3
+LOCAL_CFLAGS += -D_GNU_SOURCE
```

Comment:
Is this really needed with the changes in #173 ? If, yes, then
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`